project_code/__openerp__.py

```diff
@@ -33,16 +33,16 @@
                "images/screenshot3.png"],
     "category": "Project Management",
     "complexity": "easy",
-    "depends": ["project"],
+    "depends": ["project", "analytic_code"],
     "description": """
```

Comment:
It would be nice to put the description in a README.rst file (see https://github.com/acsone/acsone-addons/tree/8.0/multi_company_consolidation)